Makefile.dune

```diff
@@ -34,10 +34,10 @@ watch: voboot
 release: voboot
 	dune build $(DUNEOPT) -p coq
 
-apidoc:
+apidoc: voboot
         # Ugly workaround for https://github.com/ocaml/odoc/issues/148
 	mv checker/dune checker/dune.disabled || true
-	dune build $(DUNEOPT) @doc
+	dune build $(DUNEOPT) @doc || mv checker/dune.disabled checker/dune && exit 1
```

Comment:
You can always put `{ }` around if you're not sure (I'm not either)

Reply:
Umm, I think what we have now doesn't work, and `{ }` gives me a syntax error, `( )` doesn't work with exit for obvious reasons. I love `sh`.